pkg/apis/externaldns/types.go

```diff
@@ -395,6 +397,7 @@ func (cfg *Config) ParseFlags(args []string) error {
 	app.Flag("service-type-filter", "The service types to take care about (default: all, expected: ClusterIP, NodePort, LoadBalancer or ExternalName)").StringsVar(&cfg.ServiceTypeFilter)
 	app.Flag("managed-record-types", "Comma separated list of record types to manage (default: A, CNAME) (supported records: CNAME, A, NS").Default("A", "CNAME").StringsVar(&cfg.ManagedDNSRecordTypes)
 	app.Flag("default-targets", "Set globally default IP address that will apply as a target instead of source addresses. Specify multiple times for multiple targets (optional)").StringsVar(&cfg.DefaultTargets)
+	app.Flag("ignore-empty-targets", "Ignore endpoints that could not resolve targets from their ingress").BoolVar(&cfg.IgnoreEmptyTargets)
```

Comment:
Please add this to the end user documentation in the appropriate places.

Reply:
Could you point me to the right place to document this further?